mlir/test/lib/Dialect/Vector/TestVectorTransforms.cpp

```diff
@@ -480,7 +480,8 @@ struct TestFlattenVectorTransferPatterns
   }
   void runOnOperation() override {
     RewritePatternSet patterns(&getContext());
-    populateFlattenVectorTransferPatterns(patterns);
+    constexpr unsigned targetVectorBitwidth = 512;
```

Comment:
Could you make this a pass option and the toggle it in the test to demonstrate how it impacts the pattern?